templates/auth/login.html

```diff
@@ -2,8 +2,8 @@
 {% from "_forms.html" import render_field_with_errors, render_field %}
 
 {% block body %}
-<h1>{{ _('Login') }}</h1>
 <form action="{{ url_for_security('login') }}" method="POST" name="login_user_form">
+  <h1>{{ _('Login') }}</h1>
```

Comment:
Just being curious, why is it better within the `<h1>`?

Reply:
Was already within a `<h1>`. Makes sense since it's the page title, doesn't it ?

Reply:
This would semantically be the form's title then

Reply:
ah ok, well login is also the form's name. same same